sdk/loganalytics/azure-mgmt-loganalytics/azure/mgmt/loganalytics/aio/operations/_tables_operations.py

```diff
@@ -144,7 +144,7 @@ async def update(
             401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError
         }
         error_map.update(kwargs.pop('error_map', {}))
-        api_version = "2020-10-01"
+        api_version = "2020-08-01"
```

Comment:
customer asks '2020-10', but the code is '2020-08'. Please check it

Reply:
api revert link : https://github.com/Azure/azure-rest-api-specs/pull/14275/files